Challenge-1/package.json

```diff
@@ -32,7 +27,8 @@
     "enable": "forta-agent enable",
     "keyfile": "forta-agent keyfile",
     "stake": "forta-agent stake",
-    "test": "jest"
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
   },
   "dependencies": {
```

Comment:
Two notes

`ethers` is not imported anywhere in the code and can be removed
`forta-agent-tools`, as it's only used for testing in this bot, can be moved to `devDependencies`